index.html

```diff
@@ -201,5 +202,58 @@ <h4>Skills</h4><br>
             </ul>
         </div> 
 </section> 
+<!--CONTACT PAGE STARTS HERE -->
+<!--FORM DESIGN STARTS HERE -->
+<section class="page">
+    <div class="gap"></div>
+    <div class="c-form">
+    <div class="f-header">
+        <h2 class="getStart">Get started</h2>
+        <p class="About">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it .</p>
+    </div>    
+    <form class="b" action="https://formspree.io/f/xnqrlgak" method="post">
+        <div class="boxForm">
+            <label class="f-name" for="name"> 
+                Your name
+                <input class="form-name-content" type="text" id="name" name="user-name" maxlength="30" required>
+            </label>
+            <label class="f-name" for="email">
+                Your email
+                <input class="form-name-content" type="email" id="mail" name="user_email" required>
+            </label>
+            <label class="form-name-content">
+                <textarea class="form-text" name="user_message" placeholder="Enter your message here..." maxlength="500" required></textarea>
+            </label>
+        </div>
+        <div class="button bottonform">
+            <button class="blackBoton textBoton" type="submit">
+                Start collaboration
+            </button>
+        </div>
+    </form>
+
+</div>
+</section> 
+
+<!--footer section-->
+
+<footer class="cfooter">
+<div>
+    <button class="blackBoton resume">
+        Get my Resume &nbsp <img class="get-my-resume-image" src="./images/Union (2).png">
+    </button>
+</div>
+<div>
+    <ul class="rsFooter" >
+        <li class="rrss"><img src="images/Vector.png" alt="link"> </li>
+        <li class="rrss"><img src="images/Vector (3).png" alt="GitHUB"> </li>
+        <li class="rrss"><img src="images/Vector (4).png" alt="Twitter"></li>
+        <li class="rrss"><img src="images/Vector (5).png" alt="LinkedIn"></li>
+        <li class="rrss"><img src="images/Vector (6).png" alt="M"></li>
+        <li class="rrss"><img src="images/Vector (7).png" alt="RabitHand"></li>
+    </ul>
+</div>
+</footer>
+</section> 
```

Comment:
- [ ] I like how well you stick to the design because your colours are absolutely the same as in the FIGMA design :clap: .However, details such as `margin`, `padding` and `space between blocks` `alignment` are essential to stick to a design template.
Kindly make sure that the alignment of the ```Get my resume``` and the `footer` is the same as in the Figma design :+1: 

Your design

![Screenshot from 2022-12-08 17-14-17](https://user-images.githubusercontent.com/80458199/206485528-4bb9f962-8881-41ca-b8f9-ee4e0f7e135d.png)

Figma Design

![Screenshot from 2022-12-08 17-23-27](https://user-images.githubusercontent.com/80458199/206485566-94d9bcc9-bc42-44de-a951-afe8473cadb8.png)